great_expectations/core/expectation_configuration.py

```diff
@@ -1382,9 +1382,11 @@ class ExpectationConfigurationSchema(Schema):
     kwargs = fields.Dict()
     meta = fields.Dict()
     ge_cloud_id = fields.UUID(required=False, allow_none=True)
-    expectation_context = fields.Nested(lambda: ExpectationContextSchema)
+    expectation_context = fields.Nested(
+        lambda: ExpectationContextSchema, required=False, allow_none=True
+    )
 
-    REMOVE_KEYS_IF_NONE = ["ge_cloud_id"]
+    REMOVE_KEYS_IF_NONE = ["ge_cloud_id", "expectation_context"]
 
```

Comment:
This value is not mandatory so we should make it optional. If it is null, let's remove it before converting to JSON. I believe this approach should be standardized for newer attributes.

Reply:
Love this - I think it will save us a lot of pain moving forward. Let's make sure to include this in the discussion on code standards.